src/openms/include/OpenMS/ANALYSIS/OPENSWATH/MRMTransitionGroupPicker.h

```diff
@@ -473,6 +478,7 @@ namespace OpenMS
           f.setMetaValue("total_mi", transition_total_mi);
         }
 
+        // TODO shouldn't this be quantifying transition?
         if (transition_group.getTransitions()[k].isDetectingTransition())
         {
           total_intensity += peak_integral;
```

Comment:
@grosenberger can you comment on this?

Reply:
Yes, that seems to be an artifact. So far all detecting transitions are also quantifying transitions in our workflows, but we should change this for potential future extensions. Thanks for spotting this!